lib/packwerk/run_context.rb

```diff
@@ -52,7 +54,32 @@ def initialize(
 
     sig { params(file: String).returns(T::Array[T.nilable(::Packwerk::Offense)]) }
     def process_file(file:)
-      file_processor.call(file)
+      # 1. file path to node
```

Comment:
it's more like file path to AST, but that contains nodes, so... you're not wrong